### DISCUSSION TITLE
Figma-to-XAML plugin preview should show x:Class First to Avoid Nasty Bugs

### DISCUSSION THREAD

**Discussion by elibroftw** [unoplatform/uno#12157] [Ideas]
When trying to be productive and copying lots of designs, a bug in the figma-xaml plugin resulted in me copying the xaml of a page I already had. The bug in question can be shown here https://github.com/unoplatform/uno/discussions/12156#discussioncomment-5797275. 

This bug would've been caught sooner if the x:Class attribute was shown without having to scroll. Just expanding the plugin window to this width takes 1 minute and you still have to scroll right every time just to confirm that the plugin selected the right page.

![image](https://user-images.githubusercontent.com/21298211/236065322-1c9aec5d-ad26-4b70-bb07-b84883ba68d8.png)

**Comment by carldebilly** [comment]
I agree it's annoying. I just added something in the backlog to improve this. Thanks for the suggestion!
